static/js/i18n.ts

```diff
@@ -28,11 +29,14 @@ export const $t = intl.formatMessage;
 export const default_html_elements = Object.fromEntries(
     ["b", "code", "em", "i", "kbd", "p", "strong"].map((tag) => [
         tag,
-        (content_html) => `<${tag}>${content_html}</${tag}>`,
+        (content_html: string) => `<${tag}>${content_html}</${tag}>`,
     ]),
 );
 
-export function $t_html(descriptor, values) {
+export function $t_html(
+    descriptor: MessageDescriptor,
+    values?: Record<string, string | ((s: string) => string)>,
```

Comment:
FormatJS types this `Record<string, PrimitiveType | FormatXMLElementFn<string, string>>`.

Reply:
Updated, though do you feel strongly this changed? I feel like it's fine as it was before.

Reply:
We definitely need to accept numbers, for example. And in general, we need `$t_html` to use the same API as `intl.formatMessage` so it can be used with FormatJS message extraction, so I do think it’s better for our types to reflect that.